cmd/checkheader/main.go

```diff
@@ -113,14 +113,14 @@ func generate(header string) (string, error) {
 	for sc.Scan() {
 		l := strings.TrimSpace(sc.Text())
 		if enum {
-			if !strings.HasPrefix(l, "}") {
-				tryConvertEnum(&b, l)
+			if !strings.HasPrefix(l, ")") {
+				tryConvertGoConst(&b, l)
 			} else {
 				enum = false
 			}
 			continue
 		}
-		if strings.HasPrefix(l, "enum {") || strings.HasPrefix(l, "typedef enum {") {
+		if l == "const (" {
```

Comment:
This `"const ( // no_check" == "const ("` check is actually what skips the const blocks, not the `strings.HasSuffix(l, "// nocheck")` check in `tryConvertGoConst`... seems confusing.

Maybe change this line to look for a `const (` prefix and lack of `nocheck` suffix, so the intent is clearer?

Reply:
Will do!